src/UserManager.ts

```diff
@@ -289,15 +311,22 @@ export class UserManager {
         if (user?.refresh_token) {
             logger.debug("using refresh token");
             const state = new RefreshState(user as Required<User>);
+            const extraHeaders: Record<string, ExtraHeader> = {};
```

Comment:
extraHeaders is now empty, thus we do not need it (i guess that is a leftover from refactoring...)